src/components/taglib/TransformHelper/handleRootNodes.js

```diff
@@ -199,9 +199,16 @@ module.exports = function handleRootNodes() {
                     hasLegacyExplicitBind = true;
                 } else {
                     if (node.hasAttribute('id')) {
+                        let id = node.getAttributeValue('id');
+
+                        if (id && id.name) {
```

Comment:
Also, please add a link to an error page with more explanation of the restriction. We are using the Wiki to host error pages: https://github.com/marko-js/marko/wiki/Error-Messages